src/platform/site-wide/user-nav/sass/user-nav.scss

```diff
@@ -50,7 +50,7 @@ span.sidelines {
   .sitewide-search-drop-down-panel-button, .sign-in-drop-down-panel-button {
     border-top: none;
     margin-right: 0;
-    padding: 0 30px 0 0;
+    padding: 8px 30px 8px 8px;
```

Comment:
definitely not 9px;

Reply:
this is a follow-up to https://github.com/department-of-veterans-affairs/vets-website/pull/16512?